tests/integration-tests/configs/pcluster3.yaml

```diff
@@ -98,4 +98,10 @@ test-suites:
       dimensions:
         - regions: ["eu-west-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: ["centos7"]
\ No newline at end of file
+          oss: ["centos7"]
+    test_update.py::test_update_imds_secured:
+      dimensions:
+        - regions: [ "us-east-1" ]
```

Comment:
does it make sense to add a china and govcloud region as well?

Reply:
I think it would be useful only if images in those regions are peculiar in a way that they change the normal interactions with iptables. Otherwise, it would be redundant.
Do you have such suspect? If you don't I'd remain with one standard region.